src/features/RecipeLibrary/types.ts

```diff
@@ -0,0 +1,14 @@
+import { Photo, User as UserType } from "__generated__/graphql";
+
+export interface RecipeCard {
+    id: string;
```

Comment:
I'd use `BfsId` here, which makes interop between the graphql and flux pieces easier, and enumerate all the places typescript knows we care.